test/routes/entry-point.test.ts

```diff
@@ -4,11 +4,9 @@ import entryPoint from '../../src/routes/entry-point';
 import runMiddleware from '../middleware';
 import createContext from '../context';
 
-const makeRequest = async (next?: Next): Promise<Response> => {
-  const context = createContext();
-
-  return runMiddleware(entryPoint(context.router), context, next);
-};
+const makeRequest = async (next?: Next): Promise<Response> => (
+  runMiddleware(entryPoint(), createContext(), next)
```

Comment:
`makeRequest` doesn't return but can still `await` on it to get a response?

Reply:
Parentheses rather than curly brackets (line above) are a return.

Reply:
ah, it's Ruby all over again :+1: